src/test/java/org/mastodon/mamut/tomancak/lineage_registration/Glasbey.java

```diff
@@ -0,0 +1,21 @@
+package org.mastodon.mamut.tomancak.lineage_registration;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+/**
+ * Glasbey Color Tables.
+ */
+public class Glasbey
```

Comment:
IntelliJ complains that this class does contain Unit-Tests, but is not named e.g. Glasbey*Test*

I would recommend removing the tests. They are not very strong anyway.

Reply:
Thank you for spotting this.